package.json

```diff
@@ -49,8 +49,7 @@
     "etl": "node -r dotenv-flow/config -r esbuild-register scripts/etl",
     "init": "lerna clean --yes && run-s bootstrap build",
     "lerna:clean": "lerna clean",
-    "lint": "eslint --cache .",
-    "lint-next": "eslint `git diff --name-only next --diff-filter ACMRTUXB`",
+    "lint": "turbo run lint --continue -- --quiet",
```

Comment:
TIL: `--continue` stops turbo from shortcuting onthe first error. This might be useful to see all the linting error happening in your PR. 

https://turbo.build/repo/docs/reference/command-line-reference/run#--continue